src/System.Management.Automation/security/CatalogHelper.cs

```diff
@@ -423,7 +423,7 @@ internal static string CalculateFileHash(string filePath, string hashAlgorithm)
             FileStream fileStream;
             try
             {
-                fileStream = File.Open(filePath, FileMode.Open, FileAccess.Read);
+                fileStream = File.OpenRead(filePath);
```

Comment:
Taking into account a nature of the issue we fix I'd prefer explicit parameters.
```suggestion
                fileStream = new FileStream(filePath, FileMode.Open, FileAccess.Read, FileShare.Read);
```

Reply:
I was trying to be consistent with the existing code base.  Get-FileHash uses this method.  If this is the method to resolve it, I would then create an Issue that these are inconsistent.

Reply:
I believe the source of this bug is the fact that no explicit arguments can be seen. Therefore, I ask to specify the arguments explicitly.